static/js/components/dashboard/FinancialAidCard.js

```diff
@@ -0,0 +1,193 @@
+// @flow
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import { Card, CardTitle } from 'react-mdl/lib/Card';
+import Icon from 'react-mdl/lib/Icon';
+
+import DateField from '../inputs/DateField';
+import type { CoursePrice } from '../../flow/dashboardTypes';
+import type { Program } from '../../flow/programTypes';
+import { courseListToolTip } from './util';
+import { formatPrice } from '../../util/util';
+import {
+  FA_STATUS_APPROVED,
+  FA_STATUS_AUTO_APPROVED,
+  FA_STATUS_PENDING_DOCS,
+  FA_STATUS_PENDING_MANUAL_APPROVAL,
+  FA_STATUS_REJECTED,
+  FA_STATUS_DOCS_SENT,
+  FA_STATUS_SKIPPED,
+} from '../../constants';
+
+const price = price => <span className="price">{ formatPrice(price) }</span>;
+
+export default class FinancialAidCard extends React.Component {
+  props: {
+    program: Program,
+    coursePrice: CoursePrice,
+    openFinancialAidCalculator: () => void,
+    documentSentDate: Object,
+    setDocumentSentDate: Function,
+  };
+
+  documentStatus() {
```

Comment:
nitpick: can you change this to `renderDocumentStatus`? easier to tell that it's a function that renders something